### ISSUE TITLE
[FEATURE REQ] Create a mempool explorere guide

### ISSUE THREAD

**Issue by federicociro** [twofaktor/minibolt#22]
### What & why

> As a Minibolt user, I want to include a Bitcoin explorer as Mempool.
The benefits of a mempool explorer include the ability to view the current number of unconfirmed transactions, the fees associated with those transactions, and the amount of time that each transaction has been waiting to be confirmed. This information can be useful for bitcoin users and miners, as it allows them to make informed decisions about transaction fees and confirmation times. Additionally, mempool explorers can be used to detect and track network congestion, which can be helpful for identifying and addressing scalability issues on the bitcoin network.

### How

I am thinking of creating a guide on how to install a Bitcoin Mempool explorer. This guide would explain the steps required to set up and configure a mempool explorer, including any software or hardware requirements, as well as any necessary configurations.

### Additional context

Mempool will be connected with Bitcoin Core through **.cookie file** and not RPC credentials
I can help with this once PR 2846 is merged into Mempool repository.
[Mempool PR 2846 by @antonilol](https://github.com/mempool/mempool/pull/2846)

**Comment by twofaktor** [comment]
I waiting for this PR to be approved: https://github.com/mempool/mempool/pull/2846

It appears to devevoper that starts to move 🔥
